Amazon.IonDotnet.Tests/Amazon.IonDotnet.Tests.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks Condition="'$(OS)' != 'Unix'">netcoreapp2.1;netcoreapp2.0;netcoreapp1.0;net45;net46</TargetFrameworks>
-    <TargetFrameworks Condition="'$(OS)' == 'Unix'">netcoreapp2.1;netcoreapp2.0</TargetFrameworks>
+    <TargetFrameworks Condition="'$(OS)' != 'Unix'">netcoreapp2.1;net45;net46</TargetFrameworks>
+    <TargetFrameworks Condition="'$(OS)' == 'Unix'">netcoreapp2.1</TargetFrameworks>
```

Comment:
Did you intend for this change to be included in this PR?

Reply:
Yes. Tests do not compile due to .netstandard1.5 being removed from IonDotnet project in a recent PR. .netcoreapp1.0 was is not compatible with .netstandard2.0.

Removing .netcoreapp2.0 is just a "while we're in there" change, since it doesn't make sense to test using a framework that is out-of-support and superseded by .netcoreapp2.1.

Reply:
I don't have a problem with the change itself, but typically we'd want an unrelated change like this to be in its own PR (or, at the very least, in its own commit) to make the history more clear.